packages/core/src/getters/scalar.ts

```diff
@@ -131,12 +133,12 @@ export const getScalar = ({
     case 'object':
     default: {
       if (enumItems) {
-        const value = `'${enumItems
-          .map((enumItem: string) =>
-            isString(enumItem) ? escape(enumItem) : `${enumItem}`,
+        const value = `${enumItems
+          .map((enumItem: unknown) =>
+            isString(enumItem) ? `'${escape(enumItem)}'` : `${enumItem}`,
           )
           .filter(Boolean)
-          .join(`' | '`)}'`;
+          .join(` | `)}`;
```

Comment:
This code can not handle array enum, or object enum.

output is like 
```ts
export const ObjectEnum = {
  '[object_Object]': [object Object],
} as const;
```

I didn't fix because not relevant the issue, but should I leave `FIXME` comment?

Reply:
up to you if you want to leave a FIXME but not necessary?

Reply:
@melloware thanks for your review, and sorry for late reply.

I don't think necessary while someone report an issue.